nested_admin/nested.py

```diff
@@ -175,12 +175,12 @@ def media(self):
             'nesting.js',)
 
         for js_file in js_files:
-            js_file_url = '%s/nesting/%s?v=%d' % (settings.STATIC_URL, js_file, version)
+            js_file_url = '%snesting/%s?v=%d' % (settings.STATIC_URL, js_file, version)
```

Comment:
No reason, we just haven't gotten around to it. You're welcome to make a pull request :-)